netci.groovy

```diff
@@ -0,0 +1,38 @@
+import jobs.generation.Utilities
+
+def project = GithubProject
+def branch = GithubBranchName
+def isPR = true
+def platformList = ['Windows_2016:WindowsServerCore-ltsc2016', 'Windows_2016:WindowsServerCore-1709']
+
+platformList.each { platform ->
+    def(hostOS, containerOS) = platform.tokenize(':')
+    def machineLabel = 'latest-docker'
+
+    if (containerOS == 'WindowsServerCore-1709') {
```

Comment:
Per https://github.com/Microsoft/dotnet-framework-docker/commit/1f612825330fd27a99d4f20c6a68745b274e6bac, this can now be updated to a single static list - ['3.5', '4.*']

Reply:
Ok. Updated as `def versionList = ['3.5', '4.*']`